src/Tribe/I18n.php

```diff
@@ -190,13 +253,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ($flags & static::COMPILE_INPUT) {
```

Comment:
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis
* :no_entry_sign: No space before closing parenthesis is prohibited
* :no_entry_sign: No space before closing parenthesis is prohibited

<sub>hash: e3272186b0e6417ce2857329d343fde0</sub>